driver-core/src/main/com/mongodb/operation/ParallelCollectionScanOperation.java

```diff
@@ -187,7 +223,11 @@ private BsonDocument getCursorDocument(final BsonDocument cursorDocument) {
     }
 
     private BsonDocument getCommand() {
-        return new BsonDocument("parallelCollectionScan", new BsonString(namespace.getCollectionName()))
+        BsonDocument document = new BsonDocument("parallelCollectionScan", new BsonString(namespace.getCollectionName()))
                .append("numCursors", new BsonInt32(getNumCursors()));
+        if (readConcern.getValue() != null) {
```

Comment:
The value for DEFAULT is null - so essentially they are saying the same. Agreed readConcern.isServerDefault() is clearer - so will update.